content/html/HTML.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "HTML: Hypertext Markup Language"
+mdn_url: https://developer.mozilla.org/en-US/docs/Web/HTML
+recipe: landing-page
+related_content: /content/related_content/html.yaml
+link_lists:
+  - title: Learn HTML
+    pages:
+      - "/content/learn/html/Introduction_to_HTML.md"
+      - "/content/learn/html/Multimedia_and_embedding.md"
+  - title: Reference
+    pages:
+      - "/content/html/reference/elements.md"
+---
+**HTML** (HyperText Markup Language) is the most basic building block of the Web. It defines the meaning and structure of web content. Other technologies besides HTML are generally used to describe a web page's appearance/presentation ([CSS](/en-US/docs/Web/CSS)) or functionality/behavior ([JavaScript](/en-US/docs/Web/JavaScript)).
+
+"Hypertext" refers to links that connect web pages to one another, either within a single website or between websites. Links are a fundamental aspect of the Web. By uploading content to the Internet and linking it to pages created by other people, you become an active participant in the World Wide Web.
+
+HTML uses "markup" to annotate text, images, and other content for display in a Web browser. HTML markup includes special "elements" such as [`<head>`](/en-US/docs/Web/HTML/Element/head), [`<title>`](/en-US/docs/Web/HTML/Element/title), [`<body>`](/en-US/docs/Web/HTML/Element/body), [`<header>`](/en-US/docs/Web/HTML/Element/header), [`<footer>`](/en-US/docs/Web/HTML/Element/footer), [`<article>`](/en-US/docs/Web/HTML/Element/article), [`<section>`](/en-US/docs/Web/HTML/Element/section), [`<p>`](/en-US/docs/Web/HTML/Element/p), [`<span>`](/en-US/docs/Web/HTML/Element/span), [`<img>`](/en-US/docs/Web/HTML/Element/img), [`<aside>`](/en-US/docs/Web/HTML/Element/aside), [`<audio>`](/en-US/docs/Web/HTML/Element/audio), [`<canvas>`](/en-US/docs/Web/HTML/Element/canvas), [`<datalist>`](/en-US/docs/Web/HTML/Element/datalist), [`<details>`](/en-US/docs/Web/HTML/Element/details), [`<embed>`](/en-US/docs/Web/HTML/Element/embed), [`<nav>`](/en-US/docs/Web/HTML/Element/nav), [`<output>`](/en-US/docs/Web/HTML/Element/output), [`<progress>`](/en-US/docs/Web/HTML/Element/progress), [`<video>`](/en-US/docs/Web/HTML/Element/video), [`<ul>`](/en-US/docs/Web/HTML/Element/ul), [`<ol>`](/en-US/docs/Web/HTML/Element/ol), [`<li>`](/en-US/docs/Web/HTML/Element/li) and many others.
+
+An HTML element is set off from other text in a document by "tags", which consist of the element name surrounded by "`<`" and "`>`". The name of an element inside a tag is case insensitive. That is, it can be written in uppercase, lowercase, or a mixture. For example, the `<title>` tag can be written as `<Title>`, `<TITLE>`, or in any other way.
+
+The articles below can help you learn more about HTML.
```

Comment:
Maybe I've been staring myself blind and now my brains fried, but the JSON output of this .md file is this:
```json
  "body": [
    {
      "type": "prose",
      "value": {
        "title": "Overview",
        "id": "overview",
        "content": "<p><strong>HTML</strong> (HyperText Markup Language) is the most bas....snip..."
      }
    },
    {
      "type": "link_list",
      "value": {
        "title": "Learn HTML",
        "content": [
          {
            "title": "Introduction to HTML",
            "mdn_url": "https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML"
          },
          {
            "title": "Multimedia and Embedding",
            "mdn_url": "https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding"
          }
        ]
      }
    },
    {
      "type": "link_list",
      "value": {
        "title": "Reference",
        "content": [
          {
            "title": "HTML elements",
            "mdn_url": "https://developer.mozilla.org/en-US/docs/HTML/Elements"
          }
        ]
      }
    }
  ]
```

Why are the link_list sections at the bottom of this? And where does it say that they should be at the bottom?

**What if my desired final web page HTML should be something like this:**

```html
<div class="content">

  <h2>HTML: Hypertext Markup Language</h2>
 
  <!-- from Prose component -->
  <p><b>HTML</b> (HyperText Markup Language) is the most basic building block of the Web.</p>

  <!-- from LinkList component -->
  <div class="link-list">
    <h3>Links: Learn HTML</h3>
    <ul>
      <li><a href="g/en-US/docs/Learn/HTML/Introduction_to_HTML">Introduction to HTML</a></li>
      <li><a href="/en-US/docs/Learn/HTML/Multimedia_and_embedding">Multimedia and Embedding</a></li>
    </ul>
  </div>

  <!-- from Prose component -->
  <p>So, check out those links and when you're done check out these other cool references below</p>

  <!-- from LinkList component -->
  <div class="link-list">
    <h3>Links: Reference</h3>
    <ul>
      <li><a href="/en-US/docs/HTML/Elements">HTML elements</a></li>
    </ul>
  </div>

  <!-- from Prose component -->
  <p>If you've read this far, you must be really interested in this. Please also check out our <a href="/en-US/docs/Web/CSS">CSS reference docs</a></p>


</div>
```

I'm not arguing that I want/need this but I just don't understand the rationale that link_list components always appear after all the other prose.

Reply:
> Why are the link_list sections at the bottom of this? And where does it say that they should be at the bottom?

Over in https://github.com/mdn/stumptown-content/issues/101#issuecomment-530500755 I've talked about three different ways of implementing landing pages:

1. as guide pages
2. as guide pages with extra directives for lists of links
3. as a special structured type

This PR implements the third of these options. The structure definition says you get an overview, then a bunch of links. If that is too restrictive and there are extra requirements for a landing page (like, being able to have text at the end, or being able to have extra text fragments in between lists of links) then we should consider how to support them.

But you'll quite quickly get to a point where it's not possible to define sensible structures for these pages, and you'll just have to go with option 2.

I do want to support legitimate requirements for authors, but I'm resistant to add complexity just because we think we might want it.

Reply:
In other words; "Here's a thing. It's not the most flexible thing in the world, but it moves us forward."
I'll take it!

Actually, this reminds us of the essence of Stumptown too: simplicity. Being able to do less isn't necessarily about lack of flexibility as much as it's a matter of easier to comprehend.